Dockerfile

```diff
@@ -52,7 +52,7 @@ ENV PATH="/usr/share/grafana/bin:$PATH" \
 WORKDIR $GF_PATHS_HOME
 
 RUN apk add --no-cache ca-certificates bash tzdata && \
-    apk add --no-cache --upgrade --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main openssl musl-utils
```

Comment:
It was pointed out that it could be risky to mix in packages from the edge channel. Why did we do this in the first place?

Reply:
Not sure, maybe due to phantomjs or some security issue. Check blame

Reply:
@marefr I found that you made the change in #19186 in order to solve vulnerabilities. I will see if I can do a scan.

Reply:
@marefr I scanned the Docker image built from /Dockerfile on this branch, trivy reports zero vulnerabilities :)

```
✗ trivy grafana/grafana:dev                                                
2020-04-30T09:25:20.048+0200	INFO	Detecting Alpine vulnerabilities...

grafana/grafana:dev (alpine 3.11.6)
===================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

Reply:
You missed another one in https://github.com/grafana/grafana/blob/04ed8801f675780efdd848fe7d8b090525d1281b/packaging/docker/Dockerfile#L31

Reply:
@J0WI thanks!